pi-gen-sources/00-teslausb-tweaks/files/teslausb_setup_variables.conf.sample

```diff
@@ -118,6 +118,10 @@ export HEADLESS_SETUP=true
 # export aws_secret_key=put_your_secretkey_here
 # export aws_sns_topic_arn=put_your_sns_topicarn_here
 
+# Uncomment if setting up Webhook notifications
+# export webhook_enabled=true
```

Comment:
please make all new global variables in the .conf file upper case

Reply:
Okay, I was just following all the other notification types

Reply:
understood. I just want to make the globals uppercase going forward (and will be converting them)